be/src/olap/olap_snapshot_converter.cpp

```diff
@@ -89,27 +93,27 @@ OLAPStatus OlapSnapshotConverter::to_olap_header(const TabletMetaPB& tablet_meta
     return OLAP_SUCCESS;
 }
 
-OLAPStatus OlapSnapshotConverter::to_tablet_meta_pb(const OLAPHeaderMessage& olap_header,
+OLAPStatus OlapSnapshotConverter::to_tablet_meta_pb(OLAPHeaderMessage* olap_header,
```

Comment:
This function name indicates filling content from olap_header. It is not a good idea to change olap_header's content.
And comment about what will be done in this function is needed.

Reply:
Because PDelta can not corresponds with RowsetMeta. 
So before calling convert_to_rowset_meta, I add delete predicate to PDelta to store delete predicate.